### DISCUSSION TITLE
Redirect Player From Bukkit

### DISCUSSION THREAD

**Discussion by dynomake** [CloudNetService/CloudNet-v3#1245] [Q&A]
Good afternoon. I need to redirect a player to another server using the `de.dytanic.cloudnet.wrapper.Wrapper` class, there is a `player player name connect server` command in Cloudnet, how do I do the same with Bukkit?

**Comment by JWeinelt** [comment]
Answer is a bit late, but maybe still helpful:
I would use the BungeechannelAPI, it's easiert than using the Wrapper, and it's also compatible with non-cloudnet systems:
https://github.com/leonardosnt/BungeeChannelApi
